src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -883,5 +942,28 @@ private static AzureEditionDetails[] GetAzureMaxSizes(IEnumerable<AzureEdition>
             }
             return sizes.ToArray();
         }
+
+        /// <summary>
+        /// Prepares database scoped configurations list
+        /// </summary>
+        /// <param name="smoDSCMetaData"></param>
+        /// <returns></returns>
+        private DatabaseScopedConfigurationsInfo[] GetDSCMetaData(DatabaseScopedConfigurationCollection smoDSCMetaData)
+        {
+            var dscMetaData = new List<DatabaseScopedConfigurationsInfo>();
+            foreach (DatabaseScopedConfiguration dsc in smoDSCMetaData)
+            {
+                // Options MAXDOP(Id = 1) and PAUSED_RESUMABLE_INDEX_ABORT_DURATION_MINUTES(Id = 25) are integer numbers but coming as string value type, need exclusion from the below conversion
```

Comment:
Would suggest creating a helper method for this - makes it easier to update and reuse. I'd also suggest using a switch statement - bit easier to read and update later on.